docs/user-guide/a2-03-wgsl-target-specific.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: user-guide
+permalink: /user-guide/wgsl-target-specific
+---
+
+WGSL specific functionalities
+=============================
+
+This chapter provides information for WGSL (WebGPU Shading Language) -specific functionalities and behaviors.
+
+
+System-Value semantics
+----------------------
+
+The system-value semantics are translated to the following WGSL code.
+
+| SV semantic name | WGSL code |
+|--|--|
+| SV_Barycentrics | *Not supported* |
+| SV_ClipDistance<N> | *Not supported* |
+| SV_CullDistance<N> | *Not supported* |
+| SV_Coverage | `@builtin(sample_mask)` |
+| SV_CullPrimitive | *Not supported* |
+| SV_Depth | `@builtin(frag_depth)` |
+| SV_DepthGreaterEqual | *Not supported* |
+| SV_DepthLessEqual | *Not supported* |
+| SV_DispatchThreadID | `@builtin(global_invocation_id)` |
+| SV_DomainLocation | *Not supported* |
+| SV_GSInstanceID | *Not supported* |
+| SV_GroupID | `@builtin(workgroup_id)` |
+| SV_GroupIndex | `@builtin(local_invocation_index)` |
+| SV_GroupThreadID | `@builtin(local_invocation_id)` |
+| SV_InnerCoverage | *Not supported* |
+| SV_InsideTessFactor | *Not supported* |
+| SV_InstanceID | `@builtin(instance_index)` |
+| SV_IntersectionAttributes | *Not supported* |
+| SV_IsFrontFace | `@builtin(front_facing)` |
+| SV_OutputControlPointID | *Not supported* |
+| SV_PointSize | *Not supported* |
+| SV_Position | `@builtin(position)` |
+| SV_PrimitiveID | *Not supported* |
+| SV_RenderTargetArrayIndex | *Not supported* |
+| SV_SampleIndex | `@builtin(sample_index)` |
+| SV_ShadingRate | *Not supported* |
+| SV_StartVertexLocation | *Not supported* |
+| SV_StartInstanceLocation | *Not suported* |
+| SV_StencilRef | *Not supported* |
+| SV_Target<N> | *Not supported* |
+| SV_TessFactor | *Not supported* |
+| SV_VertexID | `@builtin(vertex_index)` |
+| SV_ViewID | *Not supported* |
+| SV_ViewportArrayIndex | *Not supported* |
+
+
+Supported HLSL features when targeting WGSL
+-------------------------------------------
+
+The following table lists Slang's support for various HLSL feature sets, when targeting WGSL.
```

Comment:
The mention of `HLSL` doesn't seem appropriate.
Maybe we can say it without using the word `HLSL`
```
Supported features when targeting WGSL
```
Or we can replace it with Slang
```
Supported Slang features when targeting WGSL
```

Reply:
IMO, deferring to HLSL is useful if someone wants to read more about any given feature.
We haven't documented each of these feature sets ourselves, as far as I know.